src/SourceGenerators/Uno.UI.SourceGenerators/RoslynMetadataHelper.cs

```diff
@@ -157,6 +159,38 @@ public ITypeSymbol GetTypeByFullName(string fullName)
 		private static bool SourceIsTypeImplemented(INamedTypeSymbol type)
 			=> type.GetAttributes().None(a => a.AttributeClass?.GetFullyQualifiedTypeExcludingGlobal() == XamlConstants.Types.NotImplementedAttribute);
 
+		public ISymbol? FindPropertyByOwnerSymbol(INamedTypeSymbol? type, string propertyName) => _findPropertyByOwnerSymbol!(type, propertyName);
+
+		private ISymbol? SourceFindPropertyByOwnerSymbol(INamedTypeSymbol? type, string propertyName)
```

Comment:
Is this the same as `SourceFindPropertyTypeByOwnerSymbol` except that it doesn't access `.Type`?

In this case, I'd actually drop `FindPropertyTypeByOwnerSymbol` completely, and use `FindPropertyByOwnerSymbol` followed by `.Type` where `FindPropertyTypeByOwnerSymbol` was used.